### ISSUE TITLE
Is there a need to pass in options to mount()

### ISSUE THREAD

**Issue by joetidee** [jakubjanczyk/enzyme-custom-wrappers#1]
I am receiving the following error when running my jest tests on a component:

> Uncaught [Error: Invariant failed: You should not use <Link> outside a <Router>]

As per this post, do we need to be able to pass in `options` as the second parameter to mount()?

https://seabadger.io/tech/web-development/test-component-jest-enzyme-outside-router/

**Comment by jakubjanczyk** [comment]
Good catch. I didn't have use case for this, thus it's not included the `mountWithCustomWrappers` function - it might be in future version.
However, this function is high level abstraction. You can still mount component by yourself, passing any option to mount you might need. Just use function `createComponentWrapperFor` (it's an abstraction on lower level), like this:

`createComponentWrapperFor(myWrapper)(mount(<MyComponent/>, options))`

However, in case of react-router, there is also another solution. In my tests I do not pass context as options in mount, but I rather wrap my component with `MemoryRouter` (from `react-router` library) like this:

`mountWithCustomWrappers(<MemoryRouter><MyComponent/></MemoryRouter>)(myWrapper)`

It would create a context for your and you should not be getting the error you are seeing now.

Hope this solves your issue!
